packages/react/src/PricingOptions/PricingOptions.test.tsx

```diff
@@ -34,6 +34,23 @@ describe('PricingOptions', () => {
     expect(PricingOptionsEl.classList).toContain(cardsVariantClass)
   })
 
+  it('applies the correct appearance classes for each variant', () => {
+    const solidVariantº = 'PricingOptions--appearance-solid'
```

Comment:
Who's this little guy sneaking in here? 😄


```suggestion
    const solidVariant = 'PricingOptions--appearance-solid'
```

Reply:
Looks like you'd already fixed this too 🏎️